src/globals/settings/_typography.scss

```diff
@@ -11,7 +11,7 @@ h1,
   line-height: 36px;
   margin: 56px 0 0;
   @include mq($from: tablet) {
-    font-size: govuk-em(48px, $base-font-size);
+    font-size: govuk-em(40px, $base-font-size);
```

Comment:
Need to check with @davidhunter08 that this change is OK as its a change to the global frontend library.